packages/hoppscotch-backend/src/admin/admin.service.ts

```diff
@@ -51,8 +51,8 @@ export class AdminService {
 
   /**
    * Invite a user to join the infra.
-   * @param adminUID Admin's UID
-   * @param adminEmail Admin's email
+   * @param adminUID Admins UID
+   * @param adminEmail Admins email
```

Comment:
This correction is wrong if you are talking grammatically.
`adminUID` and `adminEmail` represent UID and the Email of the admin (i.e Admin's UID and Admin's Email).

This should be reverted.

Reply:
Thanks, Andrew for the clarification, I'll revert this change.